CONTRIBUTING.md

```diff
@@ -36,16 +47,18 @@ To be active in the Working Group, you can participate in any of these places. M
 place on the mailing list, but if you just want to comment on and raise issues, that's fine too.
 
 
-## Raising Issues
+### Post-IESG Process
 
-We use our [Github](https://github.com/quicwg/) issues lists to track items for discussion and
-their resolution.
+The Working Group has built consensus that is reflected in base-draft documents, which has been confirmed through the IETF Last Call and IESG review stages modulo any critical undiscovered issues. Design changes will no longer be considered unless there are severe security, deployment or implementation problems. modulo any open (or undiscovered) issues. The goal of the Post-IESG proces is to minimise changes that invalidate the accumulated consesus, which risks returning us to the pre Working Group Last Call stage of standardisation.
 
-Before filing a new issue, please consider a few things:
+In this process, all required parties will discuss each design or major editorial issue and proposed resolution (ideally based upon a Pull Request that specifies the exact changes to be made). Chairs will judge consensus, labelling the issue as `has-consensus`.
```

Comment:
I think that you want to just say "Any change will be subject to careful review and discussion, which might involve the editors, chairs, working group, and our area director."  Leaving aside the question of how much discussion, which we might vary based on what the change is.

The `has-consensus` piece can go.